### PR TITLE
Front-End: Header Component (Responsive)

### PR DESCRIPTION
**Pull request by saint-fox1**

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -4,20 +4,16 @@ export { default as AddNewWarehouse } from "./AddNewWarehouse/AddNewWarehouse";
...
 export { default as RedButton } from "./Buttons/RedButton/RedButton";
 export { default as WhiteButton } from "./Buttons/WhiteButton/WhiteButton";
-
 export { default as MainSubHeader } from "./MainSubHeader/MainSubHeader";
 export { default as WarehousesListColumns } from "./WarehousesListColumns/WarehousesListColumns";
-
 export { default as Footer } from "./Footer/Footer";
-
 export { default as AddEditSubheader } from "./AddEditSubheader/AddEditSubheader";
+export { default as Header } from "./Header/Header.js";
```
Comment (ndwilliamson123) [inline src/components/index.js:19]: The folder name is 'header' (lowercase) so this import file needs to say "./header/Header.js"

File: src/components/header/Header.scss [hunk 2]
```diff
@@ -0,0 +1,62 @@
+@use "./../../styles/partials/global" as *;
+@use "./../../styles/partials/variables" as *;
+@use "./../../styles/partials/mixins" as *;
+
+.header {
+  background-color: $graphite;
+  &__content-wrapper {
+    height: 12.5rem;
```
Comment (ndwilliamson123) [inline src/components/header/Header.scss:8]: We need some kind of extra height and/or negative margin to match the mockup. The header extends below the top of the components a little.

**Review by ndwilliamson123 — CHANGES_REQUESTED** [review]
Looks very good visually! Nice work.

Just a couple quick changes and it's good to go :)

File: src/components/index.js (hunk 1 above)
Comment (ndwilliamson123) [reply src/components/index.js:19]: Or changing the folder name to Header to match our other folder would work

Comment (saint-fox1) [reply src/components/index.js:19]: huh! totally though I fixed that. Nice catch :)

File: src/components/header/Header.scss (hunk 2 above)
Comment (saint-fox1) [reply src/components/header/Header.scss:8]: to be done in improvements ticket

**Review by ndwilliamson123 — APPROVED** [review]
Approving due to branching issue. Need to update 'header' foldername to 'Header'

**Pull request merged**